### DISCUSSION TITLE
Need help to get the drag function?

### DISCUSSION THREAD

**Discussion by Abhisheknanda1344463** [GrapesJS/grapesjs#3738] [Q&A]
Hi @artf  
I got your answer on this issue #3718 and from that I override my function and changed pixels to %.
But, now the problem is that drag functionality become too sensitive to the movement of touchpad.
I am not able to get the function  with the help of which i can make it lesser sensitive.
Attaching the video for the same 

https://user-images.githubusercontent.com/20657737/131696590-39c1f84e-3789-4eb7-bbc5-712a85559e2c.mov

**Comment by artf** [comment]
Maybe you can share your code, so people might be able to help you.
